### DISCUSSION TITLE
Suggestions for grid neutrality gauge on energy dashboard

### DISCUSSION THREAD

**Discussion by wtadler** [home-assistant/frontend#9976] [Dashboard]
**Current look:**
![Screenshot of grid neutrality gauge lovelace card](https://www.home-assistant.io/images/lovelace/energy/grid-neutrality-gauge.png)
**Current tooltip text:**
`This card represents your energy dependency. If it's green, it means you produced more energy than that you consumed from the grid. If it's in the red, it means that you relied on the grid for part of your home's energy consumption.`

In the context of the energy dashboard, this card is basically redundant with information presented directly above it in the energy distribution card, especially if suggestion #9975 is implemented. The gauge just does a few operations to tell you whether you returned more than you consumed, which you can tell by glancing at the "Grid" circle on the energy distribution card anyway. Second, it’s not immediately clear what the needle position represents. It’s relatively clear that the number displayed in the gauge is just `returned - consumed`, but what does the needle position mean? What are the units/limits of the gauge? I’ve looked at [the code](https://github.com/home-assistant/frontend/blob/67ab63f00eac805887834033e3979faf3af53871/src/panels/lovelace/cards/energy/hui-energy-grid-neutrality-gauge-card.ts#L89), so I get what it’s doing (and it seems like a reasonable thing to do), but it's still not clear to the user. For these reasons, I would suggest removing this card from the energy dashboard. I could see leaving it around as an option for other Lovelace dashboards (especially for users not using the energy distribution card).

But if this card is definitely sticking around in the energy dashboard, here are some changes I would suggest:

1. Switch the green/red colors to the purple/blue return/consumption colors used in the energy distribution and energy usage cards. That’s exactly what the gauge zones represent: the needle being in the green zone indicates net energy return (indicated in the other cards by purple) and the needle being in the red zone indicates net energy consumption (indicated in the other cards by blue). This change would therefore make the card consistent with the others. The use of green/red has two additional problems as well. First, green is also used in the energy distribution card to indicate non-fossil grid consumption, which is totally different. Green/red also has the problem, IMO, of introducing positive/negative valence, which makes the tooltip on the solar consumed gauge a bit confusing (see #9977).
2. Eliminate the yellow range. The size and position is totally arbitrary, and not mentioned in the tooltip anyway. It seems only to reinforce the positive/negative valence, which I think is a problem in the first place.
3. The tooltip is incorrect. It says that if the needle is `in the red, it means that you relied on the grid for part of your home's energy consumption.` In fact, the needle represents net usage. It could be in the green even if you relied on the grid at night, which is, of course, typical. I would suggest changing the text to the following (coupled with the previous two changes):
`This card indicates your net energy usage. If the needle is in the purple, you returned more energy to the grid than you consumed from it. If it's in the blue, you consumed more energy than you returned.`

**Comment by balloob** [comment]
This card is not redundant. It breaks out an important piece of information and makes it easy to interpreted without having to look at a single circle in another graph and do math. With that reasoning we might just throw everything in a table, drop all colors and just call anything else redundant 😉 

I think adopting the colors for return to grid / consumption from grid is a good idea that we should explore (CC @matthiasdebaat)

**Reply by wtadler** [reply]
I agree, especially as other changes [seem in the works](https://github.com/home-assistant/frontend/discussions/9949#discussioncomment-1323818) for the energy distribution card.

With regards to colors, please feel free to leave comment on #10054!

**Reply by matthiasdebaat** [reply]
Thanks for your suggestions! Just made a comment
